systemtest/src/test/java/io/strimzi/systemtest/mirrormaker/MirrorMaker2ST.java

```diff
@@ -570,6 +648,18 @@ void testMirrorMaker2CorrectlyMirrorsHeaders() {
      * This test is using the Kafka Identity Replication policy. This is what should be used by all new users.
      */
     @ParallelNamespaceTest
+    @TestDoc(
+        description = @Desc("Checks MM2 with IdentityReplicationPolicy, preserving topic names between clusters."),
+        steps = {
+            @Step(value = "Deploy source and target clusters and a scraper pod.", expected = "Test infra is ready."),
+            @Step(value = "Deploy MM2 with IdentityReplicationPolicy.", expected = "MM2 uses identity policy."),
+            @Step(value = "Produce and consume messages in source.", expected = "Source cluster works."),
```

Comment:
`to source Kafka cluster` ? WOuldn't be better to have expected result something like `Messages were successfuly produced and consumed to/from source cluster.` ?